src/Microsoft.Azure.SignalR/HubHost/IClientConnectionManager.cs

```diff
@@ -9,6 +9,8 @@ internal interface IClientConnectionManager
     {
         void AddClientConnection(ServiceConnectionContext clientConnection);
 
+        void RemoveClientConnection(string connectionId);
+
         ConcurrentDictionary<string, ServiceConnectionContext> ClientConnections { get; }
```

Comment:
Is ClientConnections still used externally?

Reply:
Yes, it's part of the interface and is used to dispatch messages to specific connections.